pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -697,7 +699,13 @@ stdenv.mkDerivation (finalAttrs: {
     export DESTDIR=/
   '';
 
-  postInstall = ''
+  installPhase = ''
+    runHook preInstall
+    meson install ${lib.optionalString onlyLibs "--tags devel,libudev,libsystemd"}
```

Comment:
This should probably go into `pkgs/development/tools/build-managers/meson/setup-hook.sh`.